src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+/**
+ * This class implements the CriteriaAudioTest class File and the getter methods for testing CriteriaAudio class.
+ *
+ * @author Limbert Alvaro Vargas Laura.
+ * @version 1.0
+ */
+public class CriteriaAudioTest {
+
```

Comment:
I guess it'd be better removing this blank line

Reply:
fixed